scripts/simex

```diff
@@ -824,6 +824,25 @@ def do_queue_kill(args):
 queue_kill_parser = queue_subcmds.add_parser('kill')
 queue_kill_parser.set_defaults(cmd=do_queue_kill)
 
+def do_queue_show(args):
```

Comment:
What happens if I run this command even though the daemon is not listening?

Reply:
I added error handling in a later commit